manifests/0000_50_cluster-platform-operator-manager_00-platformoperator.crd.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    release.openshift.io/feature-gate: TechPreviewNoUpgrade
     api-approved.openshift.io: https://github.com/openshift/api/pull/1234
-    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/self-managed-high-availability: true
```

Comment:
Does the quoted "true" matter here?

Reply:
Ah, probably. I don't want to change it from what it was before.